packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayerContainer.js

```diff
@@ -0,0 +1,70 @@
+import { inject, observer } from 'mobx-react'
+import { toStream } from 'mobx-utils'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { from } from 'rxjs'
+
+import InteractionLayer from './InteractionLayer'
+
+function storeMapper (stores) {
+  const { drawing } = stores.classifierStore
+  return {
+    drawing
+  }
+}
+
+@inject(storeMapper)
+@observer
+class InteractionLayerContainer extends Component {
+  constructor () {
+    super()
+    this.onMouseDown = this.onMouseDown.bind(this)
+    this.onMouseMove = this.onMouseMove.bind(this)
+    this.onMouseUp = this.onMouseUp.bind(this)
+  }
+
+
+  componentDidMount () {
+    const stream = from(toStream(() => this.props.drawing.mouseEventStream))
```

Comment:
I'm not very well versed in rx.js's API. Just wondering why `from` is used here instead of `fromEvent` (https://rxjs-dev.firebaseapp.com/api/index/function/fromEvent) since what we're tracking are DOM events.

Reply:
I think `fromEvent` is a shorthand for binding event listeners, so we could use that instead of the React event handlers lower down. We'd then have to do cleanup ourselves when the component gets unmounted. Also, I know React also has some optimisations under the hood, like reusing event objects; I'm not sure if Rxjs does anything similar.

Reply:
In retrospect, I'm not sure this is the most idiomatic way of doing it though...